packages/p2p/src/components/my-profile/payment-methods/payment-method-card/payment-method-card.scss

```diff
@@ -18,6 +18,10 @@
         margin: 1.6rem 0;
         padding: 0.8rem;
 
+        @include mobile {
+            margin: 1rem 0;
+        }
```

Comment:
margin to adjust height of add PM card in mobile screens to fit height:
### Before
<img width="1002" alt="Screenshot 2022-06-08 at 11 11 45 AM" src="https://user-images.githubusercontent.com/103016120/172544707-6a74f920-28eb-49d3-b7d8-f8f2128dcb08.png">

### After

<img width="462" alt="Screenshot 2022-06-08 at 2 07 55 PM" src="https://user-images.githubusercontent.com/103016120/172544744-98687909-6983-4443-88e7-af1bda8f8857.png">